echo.go

```diff
@@ -13,6 +13,13 @@ func AddEchoRoutes(router *echo.Group) {
 
 func runTests(c echo.Context) error {
 	results := Run()
+
+	if instance.db != nil {
+		// TODO do we want to alert this somehow?
+		// doesn't make sense to return an http error since we do have test results
```

Comment:
is it possible to add a suite-level log message?

Reply:
Probably. Might not get noticed but we also don't actually use this endpoint currently since it's synchronous; it could use some attention at some point.

Reply:
The problem is the lack of a logging library.